src/main/java/seedu/address/wrapper/UserInfoWrapper.java

```diff
@@ -0,0 +1,88 @@
+//@@author arnav-ag
+package seedu.address.wrapper;
```

Comment:
`wrapper` does not provide too much clarification regarding this package. Perhaps we can change to `seedu.address.github` or `seedu.address.logic.github`